recipes/eglibc/eglibc_2.18-r24943.oe.sig

```diff
@@ -1 +1,2 @@
+517cfd34b8d250dcdca0f3490fa4e9bf571ee473  svn://www.eglibc.org/branches/eglibc-2_18;rev=r24943
```

Comment:
As mentioned, we should investigate why this signature is different.  I have a copy of the old ingredient file, so I guess I should investigate it.